apps/condo/domains/billing/schema/BillingRecipient.js

```diff
@@ -45,6 +45,24 @@ const BillingRecipient = new GQLListSchema('BillingRecipient', {
             isRequired: true,
         },
 
+        bankName: {
+            schemaDoc: 'Bank name',
+            type: Text,
+            isRequired: false,
+        },
+
+        offsettingAccount: {
+            schemaDoc: 'Bank account',
```

Comment:
Could you please write a description of what does it mean?
I've looked up a term "offsetting account banking" and found following "A savings or transaction account, that's typically linked to an eligible home loan".
Is it it?

Reply:
Will add all translates to one file (iec, tin, etc ...) DOMA-2401